src/autopas/containers/verletListsCellBased/verletListsCells/PairwiseVerletNeighborList.h

```diff
@@ -0,0 +1,140 @@
+/**
+ * @file PairwiseVerletNeighborList.h
+ * @author tirgendetwas
+ * @date 03.11.20 od. 07.11.20
+ */
+
+#pragma once
+#include "autopas/containers/verletListsCellBased/verletListsCells/VerletListsCellsNeighborListInterface.h"
+#include "autopas/containers/verletListsCellBased/verletListsCells/traversals/VLCTraversalInterface.h"
+#include "autopas/utils/StaticBoolSelector.h"
+
+namespace autopas {
+template <class ParticleCell>
+class TraversalSelector;
+/**
+ * Neighbor list to be used with VerletListsCells container.
+ * Pairwise verlet lists iterates through each pair of neighboring cells
+ * and generates a neighbor list for each particle from cell1, which consists of its (potential) partners from cell2.
+ * @tparam Particle Type of particle to be used for this neighbor list.
+ * */
+template <class Particle>
+class PairwiseVerletNeighborList : public VerletListsCellsNeighborListInterface<Particle> {
+ public:
+  /**
+   * Type of the data structure used to save the neighbor lists.
+   * */
+  using listType = typename VerletListsCellsHelpers<Particle>::PairwiseNeighborListsType;
+  /**
+   * Constructor for PairwiseVerletNeighborList. Initializes private attributes.
+   * */
+  PairwiseVerletNeighborList() : _aosNeighborList{}, _particleToCellMap{}, _globalToLocalIndex{} {}
```

Comment:
you can also state these trivial initializations at the declaration of the attributes below and omit the constructor.